packages/create-plugin/src/utils/utils.npm.ts

```diff
@@ -236,3 +237,17 @@ function sortKeysAlphabetically(obj: Record<string, string>) {
       return acc;
     }, {});
 }
+
+export function projectUsesPrettier() {
+  const packageJson = getPackageJson();
+  return Boolean(packageJson.devDependencies?.prettier);
+}
+
+export function runPrettierOnConfigDir(packageManagerName: string) {
```

Comment:
We already have this https://github.com/grafana/plugin-tools/blob/2012aa1d94e7db8864c831833d40a149338b479b/packages/create-plugin/src/commands/generate/prettify-files.ts#L7

Perhaps you can use that and modify it so it uses the logic for the package manager execution?

Additionally it'd be best if you validate that we are in the correct place. i.e. `.config` actually exists.